backend/gameplay/models.py

```
@@ -166,5 +161,40 @@ class UserGameplaySettings(models.Model):
+	
+	def is_timer_running(self):
+		return self.is_running
+	
+	@database_sync_to_async
+	def async_start(self):
+		setattr(self, "is_running", True)
+		self.save()
+
+	@database_sync_to_async
+	def async_stop(self):
+		setattr(self, "is_running", False)
```

Ruff (0.8.2)

Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)